install.sh

```diff
@@ -0,0 +1,248 @@
+#!/bin/bash
+#==============================================================================
+#title           : install.sh
+#description     : This script will install OpenNet
+#                  Support Ubuntu 14.04.1, CentOS 7, Fedora 21
+#==============================================================================
+
+set -o nounset
+set -e
+
+ROOT_PATH=`pwd`
+OVS_RELEASE='2.3.1'
+MININET_VERSION='2.2.0'
+DIST=Unknown
+RELEASE=Unknown
+CODENAME=Unknown
+
+function detect_os {
+
+    test -e /etc/fedora-release && DIST="Fedora"
+    test -e /etc/centos-release && DIST="CentOS"
+    if [ "$DIST" = "Fedora" ] || [ "$DIST" = "CentOS" ]; then
+        install='yum -y install'
+        remove='yum -y erase'
+        pkginst='rpm -ivh'
+        # Prereqs for this script
+        if ! which lsb_release &> /dev/null; then
+            $install redhat-lsb-core
+        fi
+    fi
+    if which lsb_release &> /dev/null; then
+        DIST=`lsb_release -is`
+        RELEASE=`lsb_release -rs`
+        CODENAME=`lsb_release -cs`
+    fi
+
+    grep Ubuntu /etc/lsb-release &> /dev/null && DIST="Ubuntu"
+    if [ "$DIST" = "Ubuntu" ] ; then
+        install='apt-get -y install'
+        remove='apt-get -y remove'
+        pkginst='dpkg -i'
+        # Prereqs for this script
+        if ! which lsb_release &> /dev/null; then
+            $install lsb-release
+        fi
+    fi
+    echo "Detected Linux distribution: $DIST $RELEASE $CODENAME"
+
+}
+
+function mininet {
+
+    echo "Fetch Mininet"
+    cd $ROOT_PATH
+    if [ ! -d mininet ]; then
+        git clone https://github.com/mininet/mininet.git
+    fi
+
+    cd $ROOT_PATH/mininet && git checkout tags/$MININET_VERSION
+    cp $ROOT_PATH/mininet-patch/util/install.sh $ROOT_PATH/mininet/util/
```

Comment:
1. This work just that CentOS can work on the mininet.
2. Patch conditional statement about distribution support.
3. I will follow with you suggest, thx.